src/shared-components/FormView/SelectIssuerDrawer/SelectIssuerDrawer.tsx

```diff
@@ -31,69 +33,6 @@ import {
 import { FC, useContext, useMemo, useState } from 'react';
 import { TypeaheadSelect } from '@patternfly/react-templates';
 
-const createChainOftrust = async (
-  name: string,
-  namespace: string,
-  ingressDomain: string,
-) => {
-  const rootIssuer = {
-    apiVersion: 'cert-manager.io/v1',
-    kind: 'Issuer',
-    metadata: {
-      name: name + '-root-issuer',
-      namespace: namespace,
-    },
-    spec: {
-      selfSigned: {},
-    },
-  };
-  const issuerCa = {
-    apiVersion: 'cert-manager.io/v1',
-    kind: 'Certificate',
-    metadata: {
-      name: name + 'cert',
-      namespace: namespace,
-    },
-    spec: {
-      isCA: true,
-      commonName: name,
-      dnsNames: ['issuer.' + ingressDomain],
-      secretName: name + '-cert-secret',
-      privateKey: {
-        algorithm: 'ECDSA',
-        size: 256,
-      },
-      issuerRef: {
-        name: rootIssuer.metadata.name,
-        kind: 'Issuer',
-      },
-    },
-  };
-  const content = {
-    apiVersion: 'cert-manager.io/v1',
-    kind: 'Issuer',
-    metadata: {
-      name: name,
-      namespace: namespace,
-    },
-    spec: {
-      ca: {
-        secretName: issuerCa.spec.secretName,
-      },
-    },
-  };
-
-  return await k8sCreate({ model: CertIssuerModel, data: rootIssuer }).then(
-    async () => {
-      return await k8sCreate({ model: CertModel, data: issuerCa }).then(
-        async () => {
-          return await k8sCreate({ model: CertIssuerModel, data: content });
-        },
-      );
-    },
-  );
-};
-
 type SelectIssuerDrawerProps = {
```

Comment:
There's an issue there, I've introduced a regression. I need to investigate before merging the PR.